### DISCUSSION TITLE
add a convar to allow players to apply force and setPos themselves etc

### DISCUSSION THREAD

**Discussion by DecVes** [wiremod/wire#2715] [Suggestions]
i am making an e2 right now and i just realized you cant applyforce or setpos to the owner of the entity, this would be great for people
trying to make a parkour system and things like that!

but **it needs limitations as not being able to do this on any other players**, they're probably more limitations to not lag servers etc but im not too sure what but the limitation i suggested is aa **must**.

oh and i know vurv will see this but if you have any other ideas to add so you can set your angle or get the player angle or something like that feel free to add it in, the more the merrier.

**Comment by Denneisk** [comment]
Possible duplicate of #2676

**Comment by DecVes** [comment]
> Possible duplicate of #2676

yeah it is me and vurv talked about this idea and ill still keep this one up so he can see it
